src/coll/options.rs

```diff
@@ -134,150 +322,19 @@ pub struct IndexOptions {
     pub bucket_size: Option<i32>,
 }
 
+impl IndexOptions {
+    pub fn new() -> Self {
+        Default::default()
+    }
+}
+
 /// A single index model.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct IndexModel {
     pub keys: bson::Document,
     pub options: IndexOptions,
 }
 
-/// Options for insertMany operations.
-#[derive(Clone, Default)]
```

Comment:
All of these implementations were moved to reside closer to their type definitions, for reasons mentioned above.